WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -110,16 +108,14 @@ class PrepublishingViewController: UITableViewController {
             .store(in: &cancellables)
     }
 
-    private func calculatePreferredContentSize() {
-        // Apply additional padding to take into account the navbar / status bar height
-        let safeAreaTop = UIApplication.shared.mainWindow?.safeAreaInsets.top ?? 0
-        let navBarHeight = navigationController?.navigationBar.frame.height ?? 0
-        let offset = navBarHeight - safeAreaTop
-
-        var size = tableView.contentSize
-        size.height += offset
+    override var preferredContentSize: CGSize {
+        get {
+            return tableView.contentSize
+        }
 
-        preferredContentSize = size
+        set {
+            super.preferredContentSize = newValue
+        }
```

Comment:
Rather than overriding `preferredContentSize` and treating it half like a stored property and half like a computed property, maybe override `viewDidLayoutSubviews` and assign `preferredContentSize = tableView.contentSize` ?

Reply:
Good idea! That's way better than my brute force method. Done in: f0413dcc38c301c241f1c4ec937aed20ee994040